test/ERC20RootVault.test.ts

```diff
@@ -636,6 +503,87 @@ contract<ERC20RootVault, DeployOptions, CustomContext>(
                 .addDepositorsToAllowlist([signer.address]);
         };
 
+        const setupZeroPerformanceFee = async () => {
+            var governance: IERC20RootVaultGovernance =
+                await ethers.getContractAt(
+                    "IERC20RootVaultGovernance",
+                    await this.subject.vaultGovernance()
+                );
+
+            const nftIndex = await this.subject.nft();
+            const { strategyTreasury: strategyTreasury } =
+                await governance.delayedStrategyParams(nftIndex);
+            const { strategyPerformanceTreasury: strategyPerformanceTreasury } =
+                await governance.delayedStrategyParams(nftIndex);
+
+            await governance
+                .connect(this.admin)
+                .stageDelayedStrategyParams(nftIndex, {
+                    strategyTreasury: strategyTreasury,
+                    strategyPerformanceTreasury: strategyPerformanceTreasury,
+                    privateVault: true,
+                    managementFee: BigNumber.from(3000),
+                    performanceFee: BigNumber.from(0),
+                    depositCallbackAddress: ethers.constants.AddressZero,
+                    withdrawCallbackAddress: ethers.constants.AddressZero,
+                } as DelayedStrategyParamsStruct);
+            await sleep(86400);
```

Comment:
it will be more clear if you use await sleep(this.governanceDelay) instead of magic constant